ztp/ztp-ran-policy-generator/sourcePolicies/ConfigMapRTOS.yaml

```diff
@@ -0,0 +1,41 @@
+---
```

Comment:
Why do we have this ? The PAO would apply the rt kernel

Reply:
just old stuff,  will remove it